src/ports/termux/index.ts

```diff
@@ -0,0 +1,29 @@
+import { AuraAPI } from 'core'
+import { resolve } from 'path'
+
+export async function TermuxPort(Aura: AuraAPI) {
+  const { createPort, createReadme, colorSchemes, constants } = Aura
+  const templateFolder = resolve(__dirname, 'templates')
+  const { info } = constants
+
+  const portName = 'Termux'
+  const version = '1.0.0'
+  const previewURL = `https://github.com/${info.author.username}/assets/blob/master/images/${info.slug}/aura-termux-preview.png?raw=true`
+
+  await createPort({
+    template: resolve(templateFolder, `${info.slug}.properties`),
+    replacements: {
+      ...colorSchemes.dark,
+      ...info,
+    },
+  })
+
+  await createReadme({
+    template: resolve(templateFolder, 'README.md'),
+    replacements: {
+      portName,
+      version,
+      previewURL,
+    },
+  })
+}
```

Comment:
Your port uses dark and soft colors, you must pass both colorSchemes but to make this working in the same file, you have to rename the object keys from colorSchemes.darkSoft. Update the code to:
```ts
import { AuraAPI } from 'core'
import { resolve } from 'path'

export async function TermuxPort(Aura: AuraAPI) {
  const { createPort, createReadme, colorSchemes, constants } = Aura
  const templateFolder = resolve(__dirname, 'templates')
  const { info } = constants

  const portName = 'Termux'
  const version = '1.0.0'
  const previewURL = `https://github.com/${info.author.username}/assets/blob/master/images/${info.slug}/aura-termux-preview.png?raw=true`

  await createPort({
    template: resolve(templateFolder, `${info.slug}.properties`),
    replacements: {
      ...colorSchemes.dark,
      ...withDarkSoftScheme(colorSchemes),
      ...info,
    },
  })

  await createReadme({
    template: resolve(templateFolder, 'README.md'),
    replacements: {
      portName,
      version,
      previewURL,
    },
  })
}

function withDarkSoftScheme(colorSchemes: AuraAPI['colorSchemes']) {
  return Object.entries(colorSchemes.darkSoft).reduce(
    (acc, [key, value]) => {
      return {
        ...acc,
        [`${key}Soft`]: value,
      }
    },
    {}
  )
}
```

Reply:
All done